cfme/infrastructure/provider/virtualcenter.py

```diff
@@ -1,5 +1,16 @@
 from mgmtsystem.virtualcenter import VMWareSystem
 from . import Provider
+from cfme.web_ui import Form, Input
+from cfme.common.provider_endpoint import ProviderEndpoint
+
+
+default_endpoint_form = Form(
```

Comment:
That said, one should still be able to initialize an endpoint using k:v pairs and should be able to query the class for supported values easily.